Dashboard/Views/StatusView.axaml

```diff
@@ -1,8 +1,246 @@
+
+        <Border Grid.Row="1" Grid.Column="1" Background="#d9d9d9" Height="870" Margin="0,20,0,0">
+            <!-- TabControl for AV Data, GPS, and RES -->
+            <TabControl>
+                <!-- AV Data Tab -->
+                <TabItem Header="AV Data">
+                    <Grid>
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="Auto" />
+                            <RowDefinition Height="Auto" />
+                            <RowDefinition Height="Auto" />
+                        </Grid.RowDefinitions>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="*" />
+                            <ColumnDefinition Width="*" />
+                        </Grid.ColumnDefinitions>
+                        <!-- AV Data Tab Content -->
+                        <StackPanel Orientation="Horizontal" Width="1200">
+                            <!-- Motor Moment Percentage -->
+                            <Grid Margin="5">
+                                <Grid.RowDefinitions>
+                                    <RowDefinition Height="Auto" />
+                                    <RowDefinition Height="Auto" />
+                                </Grid.RowDefinitions>
+                                <Grid.ColumnDefinitions>
+                                    <ColumnDefinition Width="*" />
+                                </Grid.ColumnDefinitions>
+                                <TextBlock Text="Motor Moment (%):" FontSize="24" Grid.Row="0" HorizontalAlignment="Center" Margin="0,15,0,0" />
+                                <StackPanel Orientation="Horizontal" Grid.Row="1" HorizontalAlignment="Center">
+                                    <StackPanel Margin="10">
+                                        <Slider Minimum="0" Maximum="100" Value="10" Orientation="Vertical" Height="600" Margin="10" IsEnabled="False" />
+                                        <TextBlock Text="Target" VerticalAlignment="Center" FontSize="24" Margin="10" />
+                                        <TextBlock Text="10%" Background="White" Padding="10,0" TextAlignment="Center" VerticalAlignment="Center" FontSize="24" Margin="10" />
+                                    </StackPanel>
+                                    <StackPanel Margin="10">
+                                        <Slider Minimum="0" Maximum="100" Value="13" Orientation="Vertical" Height="600" Margin="10" IsEnabled="False" />
+                                        <TextBlock Text="Actual" FontSize="24" Margin="10" />
+                                        <TextBlock Text="13%" Background="White" Padding="10,0" TextAlignment="Center" FontSize="24" Margin="10" />
+                                    </StackPanel>
+                                </StackPanel>
+                            </Grid>
+                            
+                            <StackPanel Orientation="Vertical" Margin="5">
```

Comment:
for AMI and EBS, could you wrap these in one grid that defines 2 columns and 2 rows, that way the right sides line up (the text background is out of line)

Reply:
![image](https://github.com/user-attachments/assets/f5c1a7e6-7dac-4dd3-aa7e-940917fc14bd)
Is this ok or do you want it to take up more of the width to the right?

Reply:
I think this looks better, are you able to ensure that the label -> value are lined up vertically also?

Most noticable for the brake values:
![image](https://github.com/user-attachments/assets/df2bc946-d432-4c0d-a79d-c66c8e172b5b)

Reply:
Should be fixed in newest push